slides/week1/week1.org

```diff
@@ -126,8 +137,12 @@ Run some demos in your terminal! Ideally you should use each of these commands a
 | =mkdir= | Make a new directory |
 | =rm= | Remove a file |
 | =rmdir= | Remove a directory |
+| =echo= | Echoes back anything you pass |
```

Comment:
Using the word in the definition. Perhaps something like 

 echo - Write arguments to the standard output